app/scripts/components/data-catalog/index.tsx

```diff
@@ -155,153 +182,209 @@ function DataCatalog({ datasets }: DataCatalogProps) {
+            setClearedTagItem={setClearedTagItem}
+            allSelected={allSelectedFilters}
+          />
+          <CatalogWrapper>
+            {renderTags}
+            {datasetsToDisplay.length ? (
+              <Cards>
+                {datasetsToDisplay.map((d) => {
+                  const topics = getTaxonomy(d, TAXONOMY_TOPICS)?.values;
+                  const allTaxonomyValues = getAllTaxonomyValues(d).map((v) => v.name);
+                  return (
+                    <li key={d.id}>
+                      <Card
+                        cardType='horizontal-info'
+                        tagLabels={allTaxonomyValues}
+                        overline={
+                          <CardMeta>
+                            <DatasetClassification dataset={d} />
+                            <CardSourcesList
+                              sources={getTaxonomy(d, TAXONOMY_SOURCE)?.values}
+                              rootPath={DATASETS_PATH}
+                              onSourceClick={(id) => {
+                                onAction(Actions.TAXONOMY_MULTISELECT, {
+                                  key: TAXONOMY_SOURCE,
+                                  value: id
+                                });
+                                browseControlsHeaderRef.current?.scrollIntoView();
                               }}
-                            >
-                              Updated <time dateTime='2023-01-01'>X time ago</time>
-                            </Link> */}
-                            </CardMeta>
-                          }
-                          linkLabel='View more'
-                          linkTo={getDatasetPath(d)}
-                          title={
-                            <TextHighlight
-                              value={search}
-                              disabled={search.length < 3}
-                            >
-                              {d.name}
-                            </TextHighlight>
-                          }
-                          description={
-                            <TextHighlight
-                              value={search}
-                              disabled={search.length < 3}
-                            >
-                              {d.description}
-                            </TextHighlight>
-                          }
-                          imgSrc={d.media?.src}
-                          imgAlt={d.media?.alt}
-                          footerContent={
-                            <>
-                              {topics?.length ? (
-                                <CardTopicsList>
-                                  <dt>Topics</dt>
-                                  {topics.map((t) => (
+                            />
+                            <VerticalDivider variation='light' />
+                            {/* TODO: Implement modified date: https://github.com/NASA-IMPACT/veda-ui/issues/514 */}
```

Comment:
(Not introduced by this PR but) might be time to just get rid of this and close this issue